### PR TITLE
Created DataCellAnalaysis page #18

### PR DESCRIPTION
**Pull request by laithAwad**
Created the page view, the routes, and the necessary adjustments in different files, I have also changed the way the verification of the file uploaded was handled, and that led to adding a function to ResearchPaperAnalysis file to make all the pages work in a similar manner. The page will allow the upload of a png file only and will show an error message if a wrong file was being uploaded or no file selected when clicking the upload button. The result of upload now shows a sample page that will later be changed when we introduce image processing and can actually process the png file uploaded and provide the cell analysis.

### CODE REVIEW

File: Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { Box, Typography, useTheme } from '@material-ui/core'
+import React, { useState } from 'react'
+
+import { DatasetUploadView } from '../DatasetUpload/DatasetUploadView'
+import { FileUploadForm } from '../DataCell/FileUploadForm'
+import { IDatasetModel } from '../../Models/Datasets/IDatasetModel'
+import Loader from "react-loader-spinner"
+import { exampleExportDatasetModel } from '../../Models/Datasets/IDatasetModel'
+
+interface IProps {
```
Comment (GD-Dheer) [inline Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx:10]: Why is this empty?

File: Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+const fileFormat = 'image/png'
+
+export const DataCellAnalysisView = (props: IProps) => {
+
+  const [analyzedDataset, setAnalyzedDataset] = useState<IDatasetModel>(null)
+  const [isProcessingDataCell, setIsProcessingDataCell] = useState(false)
+
+  const handleSubmit = async (dataCell: File): Promise<void> => {
```
Comment (GD-Dheer) [inline Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx:21]: What is the this type File? Where does it come from

File: Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    //todo make an actual API call providing a data cell 
+    return new Promise(resolve => {
+      setTimeout(() => {
+        resolve(exampleExportDatasetModel)
+      }, 2000)
+    })
+  }
+  const validatePNG = (file: File) => {
+    /// console.log(file + " -------- ") [object file]
+    /// console.log(file.type) image/png
```
Comment (GD-Dheer) [inline Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx:42]: If we dont need these consoles, can we get rid of them

**Review by GD-Dheer — APPROVED** [review]
LGTM. Not really sure yet how we are planning to support this feature but at least the frontend work is done!

**Review by ratulkai93 — APPROVED** [review]
looks good mostly, just remove maybe the unused comments/console logs.
good work!

**Pull request merged**